[2024-03-11T14:00:13.094+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v3.second_task scheduled__2021-08-07T00:00:00+00:00 [queued]>
[2024-03-11T14:00:13.315+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v3.second_task scheduled__2021-08-07T00:00:00+00:00 [queued]>
[2024-03-11T14:00:13.327+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 6
[2024-03-11T14:00:13.690+0000] {taskinstance.py:2214} INFO - Executing <Task(BashOperator): second_task> on 2021-08-07 00:00:00+00:00
[2024-03-11T14:00:13.852+0000] {standard_task_runner.py:60} INFO - Started process 2837 to run task
[2024-03-11T14:00:14.049+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v3', 'second_task', 'scheduled__2021-08-07T00:00:00+00:00', '--job-id', '717', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp80ad4_bq']
[2024-03-11T14:00:14.090+0000] {standard_task_runner.py:88} INFO - Job 717: Subtask second_task
[2024-03-11T14:00:14.604+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v3.second_task scheduled__2021-08-07T00:00:00+00:00 [running]> on host 26e63b1d0d39
[2024-03-11T14:00:15.198+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v3' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2021-08-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-08-07T00:00:00+00:00'
[2024-03-11T14:00:15.218+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-11T14:00:15.226+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and I ran after task1']
[2024-03-11T14:00:15.431+0000] {subprocess.py:86} INFO - Output:
[2024-03-11T14:00:15.439+0000] {subprocess.py:93} INFO - hey, I am task2 and I ran after task1
[2024-03-11T14:00:15.461+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-11T14:00:15.613+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=second_task, execution_date=20210807T000000, start_date=20240311T140013, end_date=20240311T140015
[2024-03-11T14:00:16.169+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-11T14:00:16.336+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
